workflows/database_params.go

```diff
@@ -6,16 +6,8 @@ import (
 )
 
 // DatabaseSetPassword sets a database password for an environment
-func DatabaseSetPassword(ctx *common.Context, environmentName string, serviceNameIn string, newPassword string) Executor {
+func DatabaseSetPassword(ctx *common.Context, environmentName string, serviceName string, newPassword string) Executor {
```

Comment:
How are you defaulting `serviceName` to the one in mu.yml if not provided on CLI?

Reply:
I added the workflow back in and used database_common to use the serviceName from mu.yml.